packs/feats/oceans-balm.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>A blessing of the living sea salves wounds and douses flames. Touch a willing living creature. It regains 1d8 HP and gains resistance 2 to fire for 1 minute. If it has persistent fire damage, it can attempt a flat check to remove it with especially appropriate help. The target is temporarily immune to healing from Ocean's Balm for 10 minutes.</p>\n<hr />\n<p><strong>Level (+2)</strong> The healing increases by 1d8, and the resistance increases by 1.</p>\n<p>[[/r (floor((max(1,@actor.level)-1)/2)+1)d8[healing]]]{Leveled HP}</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Item.Effect: Ocean's Balm]</p>"
+            "value": "<p>A blessing of the living sea salves wounds and douses flames. Touch a willing living creature. It regains 1d8 HP and gains resistance 2 to fire for 1 minute. If it has persistent fire damage, it can attempt a flat check to remove it with especially appropriate help. The target is temporarily immune to healing from Ocean's Balm for 10 minutes.</p><hr /><p><strong>Level (+2)</strong> The healing increases by 1d8, and the resistance increases by 1.</p>\n<p>@Damage[(floor((max(1,@actor.level)-1)/2)+1)d8[healing]]{Leveled HP}</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Item.Effect: Ocean's Balm]</p>"
```

Comment:
```suggestion
            "value": "<p>A blessing of the living sea salves wounds and douses flames. Touch a willing living creature. It regains @Damage[(floor((max(1,@actor.level)-1)/2)+1)d8[healing]] HP and gains resistance 2 to fire for 1 minute. If it has persistent fire damage, it can attempt a flat check to remove it with especially appropriate help. The target is temporarily immune to healing from Ocean's Balm for 10 minutes.</p><hr /><p><strong>Level (+2)</strong> The healing increases by 1d8, and the resistance increases by 1.</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Item.Effect: Ocean's Balm]</p>"
```